src/utils/keysend.ts

```diff
@@ -0,0 +1,29 @@
+import type { KeysendResponse } from '../types'
+
+const TAG_KEYSEND = 'keysend'
+
+export const parseKeysendResponse = (
+  data: Record<string, any>
+): KeysendResponse => {
+  if (data.tag !== TAG_KEYSEND) throw new Error('Invalid keysend params')
+  if (data.status !== "OK") throw new Error('Keysend status not OK')
+
+  if (!('customKey' in data.customData[0]) || data.customData[0]['customKey'] != '696969')
```

Comment:
for people using their own node they won't have these custom key/value. Should this check be removed?